addons/web/static/src/js/services/data_manager.js

```diff
@@ -26,6 +26,9 @@ return core.Class.extend({
      * Suggestion: could be refined to invalidate some part of the cache
      */
     invalidate: function () {
+        if (odoo.session_info && odoo.session_info.cache_hashes) {
+            odoo.session_info.cache_hashes.load_menus = Date.now();
+        }
```

Comment:
Not sure about this. Also, i'd require 'web.session' instead of accessing it through odoo